src/current/_includes/latest-release-details.md

```diff
@@ -0,0 +1,21 @@
+{% comment %}
+  This is included by v24.2+ install-cockroachdb-*.md upgrade-cockroach-version.md
+{% endcomment %}
+
+{% assign rd = site.data.versions | where_exp: "rd", "rd.major_version == page.version.version" | first %}
+{% assign latest = site.data.releases | where_exp: "latest", "latest.major_version == page.version.version" | sort: "release_date" | last %}
+
+{% assign released = false %}
+{% assign skippable = false %}
+{% if rd.release_date != "N/A" and rd.maint_supp_exp_date != "N/A" %}
+    {% assign released = true %}
+    {% if rd.asst_supp_exp_date == "N/A" %}
+        {% assign skippable = true %}
+    {% endif %}
+{% endif %}
+
+CockroachDB {{ latest.major_version }} is the latest supported supported production release.{% if skippable == true %} It is an [Innovation release]({% link releases/release-support-policy.md %}#innovation-releases) that is optional for CockroachDB {{ site.data.products.dedicated }} and CockroachDB {{ site.data.products.core }} but required for CockroachDB {{ site.data.products.serverless }}.{% else %} It is a required [Regular release]({% link releases/release-support-policy.md %}#regular-releases).{% endif %} To learn more, refer to [CockroachDB {{ latest.major_version }} Release Notes](https://cockroachlabs.com/docs/releases/{{ latest.major_version }}.html).
```

Comment:
The version isn't showing up for me in the [preview](https://deploy-preview-18778--cockroachdb-docs.netlify.app/docs/releases/#v24-2), within the first sentence. The paragraph just begins "CockroachDB is the latest..."

If everything else is working as expected, and there's just an issue with the variable, you could edit the sentence to avoid the version:

"This is the latest supported supported production release of CockroachDB."

Separate minor point: Ideally we can convert the release notes link to relative. Not necessary for this PR.

Reply:
Done